.github/copilot-instructions.md

```
@@ -0,0 +1,153 @@
+# Copilot Instructions — Adze
+
+Adze is an AST-first grammar toolchain for Rust. Rust 2024 edition, MSRV 1.92.0, 75-crate workspace.
+
+## Code Conventions
+
+### Edition and Language
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)